zulip_bots/zulip_bots/bots/followup/followup.py

```diff
@@ -23,15 +23,24 @@ def usage(self) -> str:
             called "followup" that your API user can send to.
             '''
 
+    def initialize(self, bot_handler: Any) -> None:
+        self.config_info = bot_handler.get_config_info('followup', optional=False)
+        self.stream = self.config_info.get("stream", 'followup')
+
     def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
+        # Send help message
+        if message['content'] == 'help':
+            bot_handler.send_reply(message, self.usage())
+            return
```

Comment:
Just a minor thought, that this new block could be part of the if/else block below (elif), since it's part of the same tests; then you don't need the return either :)

Reply:
Thanks for the review. I will move it to the if/else block itself. It makes more sense.